### PR TITLE
Make voucher code in notification clickable

### LINKED ISSUE #1370: Include a link in "new voucher" notification
After creating a new voucher, you are shown a notification that a voucher of this specific code was created. It would be nice to make that code clickable, leading to the voucher's detail/edit page.

### PR DESCRIPTION
**Pull request by Sohalt**
Fixes #1370

### CODE REVIEW

File: src/pretix/control/forms/vouchers.py [hunk 1]
```diff
@@ -179,9 +179,7 @@ def clean(self):
         return data
 
     def save(self, commit=True):
-        super().save(commit)
-
-        return ['item']
```
Comment (Sohalt) [inline src/pretix/control/forms/vouchers.py]: I'm not sure why this was there. Removing it does not break any tests, but I'm still a little concerned.

Comment (raphaelm) [reply src/pretix/control/forms/vouchers.py]: This looks weird, I'm not sure why it was there either.

File: src/pretix/control/views/vouchers.py [hunk 2]
```diff
@@ -240,8 +244,13 @@ def get_form_kwargs(self):
...
     def form_valid(self, form):
         form.instance.event = self.request.event
-        messages.success(self.request, _('The new voucher has been created: {code}').format(code=form.instance.code))
         ret = super().form_valid(form)
+        url = reverse('control:event.voucher', kwargs={
+            'organizer': self.request.event.organizer.slug,
+            'event': self.request.event.slug,
+            'voucher': self.object.pk
+        })
+        messages.success(self.request, _format_html_lazy(_('The new voucher has been created: <a href="{url}">{code}</a>'), code=form.instance.code, url=url))
```
Comment (raphaelm) [inline src/pretix/control/views/vouchers.py]: Translators tend to screw up HTML markup. Could we leave the translated string as-is and instead pass in the full link markup for ``{code}``?

**Comment by raphaelm** [comment]
Many thanks!

**Pull request merged**